types/make.js

```diff
@@ -51,6 +51,8 @@ function generatePrefixed(prefix) {
 				output += '\t' + writeFunction(part, 'name: string', 'void');
 			} else {
 				const type = testType(parts);
+				output += '\t' + writeFunction(part + '<I, E>', 'implementation: Macro<I, E, ' + type + 'Context>, input: I, expected: E');
```

Comment:
Can you put the macro declarations after the original declarations? Editors will then first show the macro-less declaration, and that is probably the most used one.